web/templates/index.html

```diff
@@ -31,7 +31,7 @@ <h5 class="card-subtitle mb-3">
                                         <label class="text-muted" for="concept"> Choose a language concept:</label>
                                         <select class="custom-select" id="concept" name="concept">
                                             {% for k, v in structures.items %}
-                                            <option value="{{ k }}">{{ v }}</option>
+                                            <option value="{{ k|linebreaksbr }}">{{ v|linebreaksbr }}</option>
```

Comment:
Since these are <options>, the <br> won't work here, so they should be removed. (same for lines 45, 54, 90, and 98)